test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTests.server.cs

```diff
@@ -108,6 +113,86 @@ public void GrpcAspNetCoreInstrumentationAddsCorrectAttributes(bool? enableGrpcA
             Assert.StartsWith("grpc-dotnet", activity.GetTagValue(SemanticConventions.AttributeHttpUserAgent) as string);
         }
 
+        // TO DO
+        // Test to check if GrpcAspNetCore instrumentation correctly adds the attributes when it creates a sibling activity
+        [Theory(Skip = "To be run after fixing the gRPC instrumentation issue #1740")]
+        [InlineData(null)]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void GrpcAspNetCoreInstrumentationAddsCorrectAttributesWhenItCreatesNewActivity(bool? enableGrpcAspNetCoreSupport)
+        {
+            // This CompositeTextMapPropagator along with the headers passed to the client.SayHello ensure that the instrumentation creates a sibling activity
+            Sdk.SetDefaultTextMapPropagator(new CompositeTextMapPropagator(new TextMapPropagator[]
```

Comment:
to force sibling activity create, we'd just need B3Propagator alone.

Reply:
I have updated the test.